Library/Homebrew/test/cmd/commands_spec.rb

```diff
@@ -36,18 +36,18 @@
   end
 end
 
-describe Homebrew do
+describe Homebrew::CommandsCommand do
   include_context "custom internal commands"
 
   specify "::internal_commands" do
-    cmds = described_class.internal_commands
+    cmds = described_class.new.internal_commands
```

Comment:
This change had to be done to prevent these test cases from failing after the ```CommandsCommand``` class is added